src/pages/Payment/Reservation/components/BookingSchedule/index.tsx

```
@@ -0,0 +1,128 @@
+
+    const colum: ColumnInferface = {
+        count: 3,
+        width: [
+            '80px',
+            'calc((100% - 80px) / 2)',
+            'calc((100% - 80px) / 2)'
+        ],
+        headNode: [
+            ...COLUMN_NAME_LIST.map((columnName, index) => (
+                <TableHead  textAlignLeft={index !== (COLUMN_NAME_LIST.length - 1)}>
+                    <TableText color={colors.gray70}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)